abel/tools/vmi.py

```diff
@@ -475,7 +475,11 @@ class Distributions(object):
             angular dependences must be inferred from very small available
             angular ranges)
     order : int
-        highest order in the angular distributions. Even number ≥ 0.
+        highest order in the angular distributions, ≥ 0
+    odd : bool
+        include odd angular orders (is enabled automatically if **order** is
```

Comment:
...it is enabled automatically if order is odd, *otherwise is False*?

Reply:
Well, the default is indeed `False`, as can be seen in the signature. Do we need to repeat it explicitly in the description? Then this probably should be done for all other parameters as well. Or just the phrase is unclear?

Reply:
Yeah, I just wanted the description to say what the default was. Probably a good idea to have this for the other parameters as well, but since you were just changing this one in this PR, you don't need to change the others as well.

Reply:
OK, I've changed in to “include odd angular orders. By default is ``False``, but is enabled automatically if **order** is odd.” and mentioned defaults for **order** and **use_sin** too (other parameters already had or do not need them) in 9af4978. Looks good now?